src/scss/components/_progress-indicator.scss

```diff
@@ -18,6 +18,14 @@
     @include body--big;
     @include margin(-$grid-container-margin, $properties: margin-left);
 
+    // style for replacing div with a button for accessibility
+    background: none;
+    color: inherit;
+    border: none;
+    padding: 0;
+    font-family: $typography-font-family-base !important;
```

Comment:
don't think this rule is required, since the `@body` mixin sets the font-family correctly to `font-family: var(--of-typography-sans-serif-font-family);`

Reply:
In this case the 'styled-like-a-div' button kept on forcing its default text-styles and I needed !important to really set the font-family.

Reply:
hm, maybe that's a FF-quirk then?